conformance/conformance_test.go

```diff
@@ -20,6 +20,7 @@ package conformance_test
 import (
 	"testing"
 
+	v1 "sigs.k8s.io/gateway-api/apis/v1"
```

Comment:
ha maybe don't need the alias for the files that don't already

maybe we should have a lint rule to make them consistent, but that can be done separately

Reply:
>ha maybe don't need the alias for the files that don't already

goimports forces it - i'm gonna assume verify will pass. If we want to tweak it I'd say let's do this separate from this PR